src/BlockSettings.php

```diff
@@ -219,7 +219,6 @@ public function set_allowed_block_types(array|bool $allowed_block_types, object
             self::PAGE_BLOCK_TYPES,
             ! Features::is_active(self::BETA_BLOCKS_FEATURE) ? [] : self::BETA_PAGE_BLOCK_TYPES,
             ! $enform_active ? [] : [ self::P4_BLOCKS_PREFIX . '/enform' ],
-            (bool) planet4_get_option('new_ia') ? [] : [ self::P4_BLOCKS_PREFIX . '/split-two-columns' ],
```

Comment:
We need to create a follow-up ticket to remove the rest of the relevant code

Reply:
Ah, nevermind, it seems it's been done already since it wasn't moved to the theme 👍